runtime/testy/runtime.py

```diff
@@ -83,11 +83,14 @@ def nonTestModePrint(*args):
           dawn_connected = False
           continue
         elif newBadThing.event == BAD_EVENTS.ENTER_TELEOP and controlState != "teleop":
-          spawnProcess(PROCESS_NAMES.STUDENT_CODE, runStudentCode, testName, maxIter)
+          name = "teleop"
+          if testName:
+              name = testName
+          spawnProcess(PROCESS_NAMES.STUDENT_CODE, runStudentCode, name, maxIter)
           controlState = "teleop"
           continue
         elif newBadThing.event == BAD_EVENTS.ENTER_AUTO and controlState != "auto":
-          # spawnProcess(autonomous code)
+          spawnProcess(PROCESS_NAMES.STUDENT_CODE, runStudentCode, "autonomous")
```

Comment:
Based on the changes here, I still don't see whether #95 has been addressed. Could you comment on that?